zanata-maven-release

```diff
@@ -324,6 +324,6 @@ run_command sed -i -e 's|<tag>.*</tag>|<tag>HEAD</tag>|' $(find . -name pom.xml
 run_command ./mvnw "${MavenOptionArray[@]}" "${MavenTagProjects[@]}" versions:set -DnewVersion=$DevelopmentVersion
 run_command ./mvnw -N scm:checkin -Dmessage="chore(version): new snapshot version ${DevelopmentVersion}" -DpushChanges=true
 
-## Push the tags
-run_command git push --tags
+## Push the relevant tags
+run_command git push --follow-tags origin $WorkingBranch
```

Comment:
What I mean is that Maven doesn't really like it when you cut the same release more than once. I'm not surprised it confuses git clones. Rewriting history often leads to trouble.

We should avoid retagging releases. The Maven way is just to increment the version and make another release, untidy though it is. Better to be untidy than mutable!